lessons/03-express-templates/inClassExercises.md

```diff
@@ -0,0 +1,12 @@
+##In-class Exercises
+1. Finish the tutorial in the [readme](https://github.com/olinjs/olinjs/tree/master/lessons/03-express-templates-mongo) and create a webapp that uses express and handlebars to display a list of classes at Olin in your web browser. Build off of the hello world application you did for homework. Once you have completed this, show the teaching team your webapp in a browser.
+
+2. There is a lot of repetitive setup and repeated code, called boilerplate code, involved when you create a new webapp. Copy the files of the webapp you've created to a new directory called 'boilerplate'. Modify the files in this new directory to create a boilerplate webapp that you can copy and paste when you begin building a new webapp. This way, you can avoid tedious repetitive setup in the future.
```

Comment:
Or yeoman? (Which I will recommend against now, because it'll introduce a lot of *unfamiliar* boilerplate code)